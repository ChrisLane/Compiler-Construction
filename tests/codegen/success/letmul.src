main() {
    let x = 10 {
        x * x;
    };
}
