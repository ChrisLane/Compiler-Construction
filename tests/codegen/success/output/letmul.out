ld 10
st r1
mv r1, r2
mv r1, r3
mul r2, r3
st r2
mv r2, r1
ld r1
